src/data/sample.json

```diff
@@ -63,95 +84,334 @@
+      "description": "Security Procedures",
+      "createdAt": "2023-06-15T08:45:00Z",
+      "updatedAt": "2023-06-18T10:20:00Z"
+    },
+    {
+      "id": "17",
+      "name": "Product Features Management & Roadmapping",
+      "description": "Product Features Management & Roadmapping",
+      "createdAt": "2023-06-15T08:45:00Z",
+      "updatedAt": "2023-06-18T10:20:00Z"
+    },
+    {
+      "id": "18",
+      "name": "Product Testing & Feedback Management",
+      "description": "Product Testing & Feedback Management",
+      "createdAt": "2023-06-15T08:45:00Z",
+      "updatedAt": "2023-06-18T10:20:00Z"
+    },
+    {
+      "id": "19",
+      "name": "Machine Learning",
+      "description": "Machine Learning",
+     "createdAt": "2023-06-15T08:45:00Z",
+    "updatedAt": "2023-06-18T10:20:00Z"
+    },
+    
+    {
+      "id": "20",
+      "name": "Deep Learning",
+      "description": "Deep Learning",
+     "createdAt": "2023-06-15T08:45:00Z",
+    "updatedAt": "2023-06-18T10:20:00Z"
+    }
+  ],
+  
+    "interestGroups": [
+      {
+        "id": "1",
+        "name": "Machine Learning & AI",
+        "description": "Interest group focused on Machine Learning and Artificial Intelligence technologies.",
+        "dateCreated": "2024-04-11T12:00:00Z",
+        "updatedAt": "2024-04-11T12:00:00Z"
+      },
+      {
+        "id": "2",
+        "name": "Infrastructure, DevOps & Cloud Engineering",
+        "description": "Interest group covering Infrastructure, DevOps practices, and Cloud Engineering solutions.",
+        "dateCreated": "2024-04-11T12:00:00Z",
+        "updatedAt": "2024-04-11T12:00:00Z"
+      },
+      {
+        "id": "3",
+        "name": "Data Science & Data Engineering",
+        "description": "Interest group dedicated to Data Science and Data Engineering disciplines.",
+        "dateCreated": "2024-04-11T12:00:00Z",
+        "updatedAt": "2024-04-11T12:00:00Z"
+      },
+      {
+        "id": "4",
+        "name": "Database Architecture & Data Modeling",
+        "description": "Interest group focusing on Database Architecture and Data Modeling techniques.",
+        "dateCreated": "2024-04-11T12:00:00Z",
+        "updatedAt": "2024-04-11T12:00:00Z"
+      },
+      {
+        "id": "5",
+        "name": "Graphic Design & Corporate Identity",
+        "description": "Interest group centered around Graphic Design principles and Corporate Identity strategies.",
+        "dateCreated": "2024-04-11T12:00:00Z",
+        "updatedAt": "2024-04-11T12:00:00Z"
+      },
+      {
+        "id": "6",
+        "name": "User Interface Design & Experience Design",
+        "description": "Interest group specializing in User Interface Design and User Experience Design methodologies.",
+        "dateCreated": "2024-04-11T12:00:00Z",
+        "updatedAt": "2024-04-11T12:00:00Z"
+      }
+    ],
+    "messages": [
+      {
+        "id": "1",
+        "sender": "1",
+        "interestGroups": "1",
```

Comment:
I'm thinking a message will only ever be published to one interest group at a time, so we don't need the plural here? we can change `interestGroups` to `interestgroup`.

Reply:
Done